src/views/NewAnalysis/NewAnalysis.tsx

```
@@ -0,0 +1,76 @@
+        <h2>{i18n.t('new_analysis.heading')}</h2>
+        <p className={style.leadText}>{i18n.t('new_analysis.description')}</p>
+      </div>
+      <div className={style.examples}>
+        {examples.map((ex) => {
+          const text = getLocalizedExample(ex);
+          return (
+            <button
+              key={text}
+              className={style.exampleChip}
+              onClick={() => onExampleClick(text)}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)